src/App.tsx

```diff
@@ -1,60 +1,149 @@
 import './App.scss';
+import React, { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/user';
+import { Todo } from './types/todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [todosForRender, setTodos] = useState(todos);
+
+  const [title, setTitle] = useState('');
+  const [UserID, addUserId] = useState(0);
+  const [titleErr, setTitleError] = useState(false);
+  const [userErr, setUserIdError] = useState(false);
```

Comment:
```suggestion
  const [userID, addUserId] = useState(0);
  const [isTitleError, setIsTitleError] = useState(false);
  const [isUserError, setIsUserError] = useState(false);
```
Use camelCase. 
Names of the set function should be similar to the noun.
Name boolean state properly
![image](https://user-images.githubusercontent.com/96409563/200248388-910f6a84-c957-453d-94a3-b99415dc3bcc.png)